### DISCUSSION TITLE
Getting ERR_CONTENT_DECODING_FAILED for webpack chunk on IG/FB's in-app browser

### DISCUSSION THREAD

**Discussion by gustavoguichard** [vercel/next.js#23729] [Help] [answered]
### What version of Next.js are you using?

10.0.9

### What version of Node.js are you using?

^12

### What browser are you using?

Instagram / Facebook's Webview

### What operating system are you using?

Android

### How are you deploying your application?

Vercel

### Describe the Bug

My website is not being able to serve one of the script chunks (the one named `_next/static/webpack.+\.js/`) in GZIP.

Apparently on FB's Webview, the Request headers are asking for the Gzip version (`accept-encoding: gzip, deflate`) whilst in any other browser it gets the Brotli version (`accept-encoding: gzip, deflate, br` => `content-encoding: br`).

### Expected Behavior

Expected to work as in any other browser where the Script chunks are served with Brotli or Gzip with correct content types set

### To Reproduce

Just open `https://vidanatural.eco.br` in your Facebook Webview. 

To inspect it you can open any website in your Facebook Webview, go to: `chrome://inspect/#devices` and inspect that webview. In the inspector you can change the browser's URL (took my a while to found that out 😅)

It seems to be a problem with the Gzip version, what do you guys think?

### Logs from a normal browser:
<img width="853" alt="Captura de Tela 2021-04-04 às 12 38 09" src="https://user-images.githubusercontent.com/566971/113514162-71d40580-9543-11eb-9ebc-901dac9403ee.png">
<img width="986" alt="Captura de Tela 2021-04-04 às 12 38 19" src="https://user-images.githubusercontent.com/566971/113514168-74cef600-9543-11eb-8b08-f947ef6b5507.png">

### Logs from Facebook's in-App browser:
<img width="546" alt="Captura de Tela 2021-04-04 às 12 38 43" src="https://user-images.githubusercontent.com/566971/113514191-97f9a580-9543-11eb-99c2-e41921de8ef8.png">
<img width="832" alt="Captura de Tela 2021-04-04 às 12 45 33" src="https://user-images.githubusercontent.com/566971/113514200-ac3da280-9543-11eb-8752-cf9710950681.png">

This is breaking all the scripts for everyone coming from Instagram / FB  (most of my visitors).

I tried disabling compression (`compress: false` in `next.config.js`) but it does not seem to help, the content is still served compressed and the issue is still there.

I know this may not be the best place to ask for help to this problem, I'm not really sure if it is a Next or Vercel issue, but we might find some sort of bug and I hope to people facing this problem can benefit from this thread.

I appreciate any help!

**Comment by gustavoguichard** [comment, marked as answer]
It turns out it was not Next.js fault.
There's a domain redirect to my website on another server and it was caching with Gzip by default.

First Time I'm on a discussion on GH, not sure how to close this out.

Thank you guys!!

**Reply by tianyingchun** [reply]
@gustavoguichard 
how about this issue, i faced same problem it seems, my next.js website it not working on FB. webview. but it works on normal browser. Do you have any suggestions?

**Reply by tianyingchun** [reply]
i used latest next.js 12.2.5
